Resources/Prototypes/_Donevinoll/modsuit.yml

```
@@ -0,0 +1,35 @@
+- type: entity
+  parent: [ClothingBackpackERTSecurity, BaseC3SyndicateContraband]
+  id: ClothingmodsuitBloodRedHardsuit
+  name:  Blood-red Modsuit
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)